101-batch-with-keyvault-pfx-password/prereqs/prereq.azuredeploy.json

```diff
@@ -0,0 +1,74 @@
+﻿{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "certificatePrefixName": {
+      "type": "string"
+    },
+    "certificatePfxBase64": {
+      "type": "securestring"
+    },
+    "certificatePfxPassword": {
+      "type": "string"
+    }
+  },
+  "variables": {
+    "storagename": "[concat('storage', uniqueString(resourceGroup().id))]",
+    "batchname": "[concat('batch', uniqueString(resourceGroup().id))]",
+    "locationname": "[resourceGroup().location]",
```

Comment:
remove this and just use the expression throughout

Reply:
Done.